apps/concierge_site/lib/helpers/session_helper.ex

```diff
@@ -0,0 +1,54 @@
+defmodule ConciergeSite.SessionHelper do
+  @moduledoc "Common functions for user sign-in with Guardian."
+
+  import ConciergeSite.Router.Helpers
+  import Phoenix.Controller, only: [put_flash: 3, redirect: 2]
+
+  alias AlertProcessor.Model.{Trip, User}
+  alias Plug.Conn
+
+  @endpoint ConciergeSite.Endpoint
+
+  @doc "Signs in a user with Guardian and redirects to the appropriate route."
+  @spec sign_in(Conn.t(), User.t()) :: Conn.t()
+  def sign_in(conn, user) do
+    conn
+    |> ConciergeSite.Guardian.Plug.sign_in(user)
+    |> redirect(to: sign_in_redirect_path(user))
+  end
+
+  @spec sign_out(Conn.t()) :: Conn.t()
+  def sign_out(conn) do
+    redirect_to =
+      if keycloak_auth?() do
+        [
+          external:
+            URI.encode(
+              "#{System.get_env("KEYCLOAK_LOGOUT_URI")}?redirect_uri=#{page_url(conn, :landing)}"
```

Comment:
question(non-blocking): Just wondering if this works for you locally? when I try to use the logout URI from 1p I get invalid `redirect_uri` but I don't know if I'm missing extra params or should have it be a specific url, I'm currently just sending it to `https://alerts.localhost:4005/` which returns a page saying "Invalid redirect uri"

Reply:
@firestack Ah, yes, that's not obvious. First I set up `alerts.localhost` pointing to `127.0.0.1` in my `/etc/hosts` file, but it sounds like you've probably already passed that step. The other thing I had to do was configure that as an allowed redirect path in Keycloak. Are you pointing at the sandbox Keycloak? Maybe it's easier to chat in Slack to diagnose further.